.github/workflows/maven.yml

```
@@ -31,6 +31,12 @@
         java-version: '23'
         distribution: 'temurin'
         cache: maven
+    - name: Get External jars
+      env:
+        SQUID_VERSION: 0.10.1
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:68: Double quote to prevent globbing and word splitting

(shellcheck)